cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -162,7 +178,12 @@ void Run(const std::vector<string>& bag_filenames) {
         break;
       }
 
-      // TODO(damonkohler): Republish non-conflicting tf messages.
+      // TODO(damonkohler): Check if republished tf messages are in conflict
```

Comment:
This behavior needs to be guarded by a flag if conflicts are not handled. Otherwise, it's impossible to replay a bag containing conflicting tf messages.